seeds.sql

```diff
@@ -1,11 +1,2532 @@
+('A Long Obedience in the Same Direction', 'Eugene Peterson', '97800000000048', 'https://example.com/covers/a_long_obedience_in_the_same_direction.jpg', 'https://example.com/books/a_long_obedience_in_the_same_direction', '2025-02-24 18:00:00.123', 1599, 3, '{"spines": ["ch1.mp3", "ch2.mp3", "ch3.mp3"], "coverFilename": "a_long_obedience_in_the_same_direction_cover.jpg"}'),
+('God''''s Will for Your Life', 'Andrew Murray', '97800000000049', 'https://example.com/covers/god''''s_will_for_your_life.jpg', 'https://example.com/books/god''''s_will_for_your_life', '2025-02-24 19:00:00.123', 1799, 4, '{"spines": ["ch1.mp3", "ch2.mp3", "ch3.mp3"], "coverFilename": "god''''s_will_for_your_life_cover.jpg"}');
+
+INSERT INTO `book_download` (`book_id`, `idp_id`, `user_id`, `downloaded_at`) VALUES
+(43, 5, 41, '2025-02-20 00:00:00'),
+(19, 8, 28, '2025-02-17 00:00:00'),
+(14, 7, 20, '2025-02-25 00:00:00'),
+(3, 4, 37, '2025-02-24 00:00:00'),
+(40, 8, 11, '2025-02-22 00:00:00'),
+(48, 5, 11, '2025-02-17 00:00:00'),
+(10, 10, 23, '2025-02-18 00:00:00'),
+(41, 3, 10, '2025-02-16 00:00:00'),
+(13, 10, 45, '2025-02-18 00:00:00'),
+(3, 2, 6, '2025-02-23 00:00:00'),
+(27, 10, 36, '2025-02-22 00:00:00'),
+(47, 7, 9, '2025-02-23 00:00:00'),
+(15, 1, 20, '2025-02-18 00:00:00'),
+(37, 4, 50, '2025-02-18 00:00:00'),
+(49, 4, 17, '2025-02-24 00:00:00'),
+(38, 7, 37, '2025-02-18 00:00:00'),
+(12, 6, 18, '2025-02-24 00:00:00'),
+(44, 6, 30, '2025-02-16 00:00:00'),
+(12, 1, 36, '2025-02-16 00:00:00'),
+(12, 5, 43, '2025-02-25 00:00:00'),
+(39, 6, 12, '2025-02-24 00:00:00'),
+(10, 6, 14, '2025-02-21 00:00:00'),
+(24, 2, 11, '2025-02-17 00:00:00'),
+(48, 10, 20, '2025-02-19 00:00:00'),
+(36, 1, 34, '2025-02-19 00:00:00'),
+(19, 4, 12, '2025-02-25 00:00:00'),
+(35, 6, 15, '2025-02-17 00:00:00'),
+(4, 6, 45, '2025-02-16 00:00:00'),
+(43, 1, 13, '2025-02-21 00:00:00'),
+(24, 9, 38, '2025-02-25 00:00:00'),
+(6, 6, 14, '2025-02-17 00:00:00'),
+(7, 4, 19, '2025-02-24 00:00:00'),
+(11, 1, 33, '2025-02-25 00:00:00'),
+(11, 5, 29, '2025-02-21 00:00:00'),
+(3, 8, 10, '2025-02-22 00:00:00'),
+(31, 9, 22, '2025-02-25 00:00:00'),
+(15, 2, 39, '2025-02-23 00:00:00'),
+(41, 3, 46, '2025-02-17 00:00:00'),
+(16, 1, 26, '2025-02-23 00:00:00'),
+(35, 4, 19, '2025-02-18 00:00:00'),
+(38, 8, 20, '2025-02-23 00:00:00'),
+(50, 9, 18, '2025-02-19 00:00:00'),
+(29, 9, 43, '2025-02-25 00:00:00'),
+(8, 3, 3, '2025-02-25 00:00:00'),
+(7, 5, 3, '2025-02-23 00:00:00'),
+(17, 6, 29, '2025-02-17 00:00:00'),
+(21, 9, 34, '2025-02-23 00:00:00'),
+(46, 7, 29, '2025-02-21 00:00:00'),
+(11, 2, 40, '2025-02-24 00:00:00'),
+(14, 3, 2, '2025-02-20 00:00:00');
+
+INSERT INTO `book_instance` (`idp_id`, `book_id`, `user_id`, `first_given_access_at`, `version`, `expires_at`) VALUES
```

Comment:
Given that you've got an AI to generate data, have you ensured that every ID's listed are referencing a valid ID, e.g. `book_instance`'s only contains `idp_id`, `book_id` and `user_id` values in those tables?

I'm adding a comment here, but this applies to the whole file. I would rather have a small dataset with valid data, that a large dataset with invalid data